applications/luci-app-tailscale/root/usr/share/rpcd/acl.d/luci-app-tailscale.json

```
@@ -0,0 +1,19 @@
+{
+	"luci-app-tailscale": {
+		"description": "Grant access to Tailscale configuration",
+		"read": {
+			"file": {
+				"/sbin/ip -s -j ad": [ "exec" ],
+				"/sbin/logread": [ "exec" ],
```

Maybe you should limit this to reading tailscale logs only:
```suggestion
				"/sbin/logread -e tailscale": [ "exec" ],
```